modules/release-notes/pages/relnotes.adoc

```diff
@@ -1,459 +1,230 @@
-
-[#release-650]
-== Release 6.5.0 (January 2020)
-
-Couchbase Server 6.5 was released in January 2020.
-
-_Quick Links_: <<supported-platforms-650>> | <<deprecation-650>> | <<known-issues-650>> | <<fixed-issues-650>>
-
-[#changes-in-behavior-650]
+[#changes-in-behavior-660]
 === Major Changes in Behavior from Previous Releases
 
 This section notes major changes in behavior from previous releases.
 
-==== Backup and Restore
-
-===== Compression
-
-In the previous versions of Couchbase Server, value compression always occurred on the backup client. 
-
-Starting with version 6.5, `cbbackupmgr` will leverage Server compression and back up the documents as compressed whenever possible.
-If the data is compressed by default, it will be backed up as compressed.
-If the data is not compressed, it will be compressed first and then backed up when value compression is opted. This improves the performance by reducing the size of the data set to back up for transmission on the pipe and for the backup itself.
-
-==== Eventing Service
-
-* The internal class `N1qlQuery()` has been removed and replaced with internal class `N1QL()`, and the latter is not syntactically backwards compatible with former.
-
-==== Networking
-
-This release introduces the following networking changes:
-
-* The Search Service now uses port 9130 / 19130 for node-to-node traffic.
-** The Search Service also no longer uses an encrypted port unless xref:learn:clusters-and-availability/node-to-node-encryption.adoc[node-to-node encryption] is enabled.
+* Search queries from N1QL
 +
-In Couchbase Server 6.0 Enterprise Edition, the Search Service explicitly required encrypted port 18094 for xref:learn:services-and-indexes/services/search-service.adoc[scatter-gather] operations between nodes running the Search Service (even if you were still using unencrypted port 8094 for client-to-node traffic).
-Starting in Couchbase Server 6.5, the Search Service uses gRPC over port 9130 / 19130 for all node-to-node traffic, and no longer uses an encrypted port for intra-cluster communication unless xref:learn:clusters-and-availability/node-to-node-encryption.adoc[node-to-node encryption] is enabled.
-
-* The 21100-21299 range of cluster and data management ports is no longer required.
-Instead, the following ports are now required for cluster management:
-** 21100 / 21150: Node-to-node communication
-** 21200 / 21250: Node-local communication
-** 21300 / 21350: Node-local communication
+Previously, for SEARCH queries from N1QL, you could use any analyzer for queries that do not use an analyzer which are - Term, Phrase, Multiphrase, Fuzzy, Prefix, Regexp, WildCard queries. However, this caused inconsistent results between covered and non-covered queries. To ensure consistent results with covering and non-covering index queries, a keyword analyzer for queries that don't use an analyzer is mandated.
```

Comment:
The construction of the first sentence is a little odd, and it should be one sentence per line, but probably fine in context of a release note.

Reply:
Previously, for SEARCH queries from N1QL, you could use any analyzer for queries that do not use an analyzer (Term, Phrase, Multiphrase, Fuzzy, Prefix, Regexp, and WildCard queries).